samples/fabric/fabric-cli/src/helpers/fabric-functions.ts

```diff
@@ -79,13 +79,15 @@ const walletSetup = async (
       logger.error(`Identity ${userName} does not exist. Please add user in the network.\n`)
       return
```

Comment:
Can we do one thing here? if register is false, still try the enrollment (in case user is registered but wallet id file is deleted, it can fetch the id, that's what my intention was initially to do). If enrollment fails, possibly user is not even registered then log this error `Identity ${userName} does not exist. Please add user in the network.\n`.

Reply:
@sanvenDev as discussed, the command ```$ ./bin/fabric-cli user add --target-network=network1 --id=alice --secret=secrettext``` can be used to recover the wallet for a user who is already registered and wallet is lost.